common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -414,6 +414,12 @@ div.problem {
     }
   }
 
+  div.inline {
+    > span {
```

Comment:
It's an immediate child selector. It selects spans that are the immediate child of a div with class inline.

See http://www.w3schools.com/cssref/css_selectors.asp